code/datums/components/crafting/recipes.dm

```diff
@@ -217,6 +217,16 @@
 	time = 40
 	category = CAT_ROBOT
 
+/datum/crafting_recipe/Vibebot
+	name = "Vibebot"
+	result = /mob/living/simple_animal/bot/vibebot
+	reqs = list(/obj/item/light/bulb = 2,
+				/obj/item/bodypart/head/robot = 1,
+				/obj/item/assembly/prox_sensor = 1,
+				/obj/item/toy/crayon = 1)
```

Comment:
no robot arm needed?

Reply:
Nope, it uses a robot head instead. If you look at the sprite, there's a head in there (which I know is supposed to be from an Ethereal shut up)